roles/etcd/tasks/install-kubeadm.yml

```
@@ -0,0 +1,20 @@
+- name: Be sure firewalld is disabled
```

Let's try to be compliant with ansible-lint when we can
```suggestion
---
- name: Be sure firewalld is disabled
```